internal/operands/template-validator/reconcile.go

```diff
@@ -89,55 +89,60 @@ const (
 )
 
 func reconcileClusterRole(request *common.Request) (common.ResourceStatus, error) {
-	return common.CreateOrUpdateClusterResource(request,
-		common.AddAppLabels(request.Instance, operandName, operandComponent, newClusterRole()),
-		func(newRes, foundRes controllerutil.Object) {
+	return common.CreateOrUpdate(request).
+		ClusterResource(newClusterRole()).
+		WithAppLabels(operandName, operandComponent).
+		UpdateFunc(func(newRes, foundRes controllerutil.Object) {
 			foundRes.(*rbac.ClusterRole).Rules = newRes.(*rbac.ClusterRole).Rules
-		})
+		}).Reconcile()
```

Comment:
Maybe make the formatting consistent, right now sometimes the reconcile is on this line, sometimes on its own one.
I don't really have a preference to either one, but just so it's consistent.